tests/cmds/conftest.py

```diff
@@ -56,26 +59,12 @@ def cli_state_without_user(sdk_without_user, cli_state):
 
 
 @pytest.fixture
-def bad_request_for_user_already_added(mocker):
-    resp = mocker.MagicMock(spec=Response)
-    resp.text = "User already on list"
-    return _create_bad_request_mock(resp)
-
-
-@pytest.fixture
-def generic_bad_request(mocker):
```

Comment:
No longer need to worry about the test conditions involving generic bad requests